### DISCUSSION TITLE
"Fatal error" when running "make" command

### DISCUSSION THREAD

**Discussion by EdSheltonUKAEA** [idaholab/moose#21562] [Q&A General] [answered]
I have tried to do the recommended regular update using the series of commands here: 
https://mooseframework.inl.gov/getting_started/installation/index.html

After starting to run the "make -j4" command, the update then crashes with some error messages like this:
```pre
> In file included from /home/eshelton/projects/moose/modules/ray_tracing/build/header_symlinks/RayTracingObject.h:12,
>                  from /home/eshelton/projects/moose/modules/ray_tracing/src/base/RayTracingAttributes.C:13:
> /home/eshelton/projects/moose/modules/ray_tracing/build/header_symlinks/RayTracingStudy.h:26:10: fatal error: libmesh/elem_side_builder.h: No such file or directory
>    26 | #include "libmesh/elem_side_builder.h"
>       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Any suggestions what might be causing this and how I can fix it?

Many thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

I recall this error from a while ago, your libmesh is out of date. 
If you are running a conda/mamba build you will want to `mamba update --all`

if that does not get you a moose-libmesh package from June, you will want to try again with `mamba update moose-libmesh` or simply wipe the environment and re-install.

Guillaume

**Reply by EdSheltonUKAEA** [reply]
Yes, thanks.  I think I may have fixed it using the advice here: 
https://github.com/idaholab/moose/discussions/19532
